index.html

```diff
@@ -0,0 +1,83 @@
+		  <span class="upload-button"></span>
+		  <span class="history-button"></span>
+		  <span class="help-button"></span>
+		  <button class="signin-button">Sign In</button>
+		</div>
+      </div>
+      <div class="content">
+        <div class="video-container">
+			<div class="video"><iframe width="560" height="315" src="https://www.youtube.com/embed/C-gEQdGVXbk" frameborder="0"
+          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
+		  </div>
+			<div class="title">
+			<h2>10 Python Tips and Tricks</h2>
+			<div class="author-info">
+				<img class="avatar" src="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAMCAgUCCAMNAwMEAwkEBQgFBQQQBQoHBxAIDAYMDAsKCwoNDhIQDQ4RDgsLEBYJERMUFQwVCA8XGBYUDhELDAoBAwQEBgUGCgYGCBQNCwsUExQODhANFBANFA0NFBAODRQJCxMJDRAUCw4LFAoLFA0UFBQNCAgICAgIDgkICAgICP/AABEIADAAMAMBEQACEQEDEQH/xAAaAAEAAwADAAAAAAAAAAAAAAAABAUHAQIG/8QAKhAAAQMDAQcDBQAAAAAAAAAAAQACAwQFEQYHEhMUITFBMjV0CFFhcbP/xAAbAQEAAgMBAQAAAAAAAAAAAAAAAgUDBAYHAf/EACkRAAEDAQUHBQAAAAAAAAAAAAABAgMEBREhMZETMzRScZLREiJDUcH/2gAMAwEAAhEDEQA/AJC8POMCAIAgCAIAgCAu4dD3WoDDFp+6Ste0OY8UUhaQR0IO71H5WylNMqXpGvavgybN/Kuhy/Ql2iDi/T11YGjJdyUoGPv6UWlmT417V8DZv5V0KNaxjCAIAgNu2L7Zb/X1+lYKi/TyQOqIqY03DYBwwMBud3PYDyuls20al08cSye29Euwy0LGnqJFe1quwJW2/bPf7HctSw0N+np4Y5eEyn4cZG6YRkZLc+T58qdp2jUx1EkbZLm/WGWhKoqJGyOajsDB1y5WBAEAQHuthfvGk/nsVpZfFx9TZpt63qSvqE961T8kfxap2txcnX8QlVb1xnaqDUCAIAgJ9ivlRpqaklts3LS0kglhl3Guw4djhwIP6IWWKV0T0exblTIm1ytX1Jmh21DqGp1VNVS3So5qWqdvzTbjWZOAPS0ADoB2C+zSvmesj1vVc1DnK9fU7NSuWEgEAQBAEAQBAEB//9k=" />
+				<span>Corey Schafer</span>
+				<button>Suscribe</button>
+			</div>
+			<div class="views">50,016 views</div>
+			<div class="video-actions">
+			<button type="button" title="Add to"><span class="add-to-button"></span><span>Add to</span></button>
+			<button type="button" title="Share"><span class="share-button"></span><span>Share</span></button>
+			<button type="button" title="More"><span class="more-button"></span><span>More</span></button>
+			<div class="up-down-votes">
+			<button type="button" title="Liked"><span class="like-button"></span><span>359</span></button>
+			<button type="button" title="Disliked"><span class="dislike-button"></span><span>14</span></button>
+			</div>
+			</div>
+			</div>
+			<div class="description">
+			<span>Published on Dec 20, 2020</span>
+			<p>
+			This video is sponsored by Skillshare. Be one of the first 500 people to sign up with this link and get 2 free months of Skillshare Premium:
+https://skl.sh/coreyschafer
+In this Python Programming video, we will be going over 10 tips and tricks for writing Pythonic code.
+			</p>
+			</div>
+        </div>
+      </div>
+	  <div class="sidebar">
+          <ul class="related-videos">
+            <li class="related-video">
+              <img src="./img/rel_video.jpg" />
+              <div class="rel-video-info">
+              <h4>Python Tutorial: Iterators and Iterables - What Are They</h4>
```

Comment:
I see what you're going for! :+1:. Please note that different HTML elements have different hierarchies and as such cannot be nested inside others as a child element. I understand that this can be tough to memorize, an easy way to make sure you're writing semantically correct code would be to use this [validator](https://validator.w3.org) to test your markup. Please give it a whirl.